proto/viam/app/v1/app.proto

```diff
@@ -122,10 +153,96 @@ message Organization {
   google.protobuf.Timestamp created_on = 3;
 }
 
+message OrganizationMember {
+  string user_id = 1;
+  string email = 2;
+  google.protobuf.Timestamp date_added = 3;
+}
+
 message ListOrganizationsResponse {
   repeated Organization organizations = 1;
 }
 
+message OrganizationInvite {
+  string invite_id = 1;
```

Comment:
Do we need to expose invite_id? all operations receive `email` + `org_id` and lookup the invite based on that. So not sure if invite_id is useful at all to return.

Reply:
IIRC from our design talk, we decided to remove the requirement for invite IDs and instead will rely on `email` + `org_id` for uniqueness/lookup. I am ok with removing it.